package.json

```diff
@@ -143,19 +148,65 @@
     "npm": "3.10.10"
   },
   "scripts": {
```

Comment:
I've just done it, but I'm not sure what's better: alphabetically, or "grouped" by workflow, similarly how the `Makefile` was structured.